vowpalwabbit/vw.sln

```diff
@@ -509,6 +511,22 @@ Global
 		{E86A833A-514D-424D-9FE7-154225031662}.Release|x64.Build.0 = Release|x64
 		{E86A833A-514D-424D-9FE7-154225031662}.Release|x86.ActiveCfg = Release|Win32
 		{E86A833A-514D-424D-9FE7-154225031662}.Release|x86.Build.0 = Release|Win32
+		{47259EFF-C4C2-4F8D-AE67-1FA6F6D5D9FE}.Debug|Any CPU.ActiveCfg = Debug|Win32
+		{47259EFF-C4C2-4F8D-AE67-1FA6F6D5D9FE}.Debug|x64.ActiveCfg = Debug|x64
+		{47259EFF-C4C2-4F8D-AE67-1FA6F6D5D9FE}.Debug|x64.Build.0 = Debug|x64
+		{47259EFF-C4C2-4F8D-AE67-1FA6F6D5D9FE}.Debug|x86.ActiveCfg = Debug|Win32
+		{47259EFF-C4C2-4F8D-AE67-1FA6F6D5D9FE}.Debug|x86.Build.0 = Debug|Win32
+		{47259EFF-C4C2-4F8D-AE67-1FA6F6D5D9FE}.DebugLeakCheck|Any CPU.ActiveCfg = Release|x64

```

Comment:
Do we really need to commit a solution named `DebugLeakCheck`. It sounds scary..

Reply:
Scary I agree, it's gone

Reply:
Actually, it seems to be an intentionally added configuration by Markus so I think I think it should stay